extensions/vscode/src/copySettings.ts

```diff
@@ -45,8 +45,32 @@ async function copyVSCodeSettingsToPearAIDir() {
     await fs.promises.mkdir(pearAIDevSettingsDir, { recursive: true });
     await fs.promises.mkdir(pearAIDevExtensionsDir, { recursive: true });
 
-    const itemsToCopy = ['settings.json', 'keybindings.json', 'snippets', 'sync', 'globalStorage/state.vscdb', 'globalStorage/state.vscdb.backup'];
     
+    // Special handling for copying settings.json to filter out layout settings
+    const settingsSource = path.join(vscodeSettingsDir, 'settings.json');
+    const settingsDestination = path.join(pearAIDevSettingsDir, 'settings.json');
+    try {
+        if (await fs.promises.access(settingsSource).then(() => true).catch(() => false)) {
+            const settingsContent = await fs.promises.readFile(settingsSource, 'utf8');
+            const settings = JSON.parse(settingsContent);
+            
+            // Filter out layout-related settings
+            const filteredSettings = Object.fromEntries(
+                Object.entries(settings).filter(([key]) => !key.includes('workbench.') && !key.includes('window.'))
+            );
+            
+            await fs.promises.writeFile(
+                settingsDestination, 
+                JSON.stringify(filteredSettings, null, 2)
+            );
+        }
+    } catch (error) {
+        console.error(`Error copying settings.json: ${error}`);
+    }
```

Comment:
we should import user's layout related settings, your change in activate.ts should fix the issue we have with the sidebar appearing in the left

Reply:
@jpan8866 thats the thing, my change in layout settings moves pear to right in secondary side bar, and then the settings are copying async in background, and when it finishes, the settings.json get overwritten and new imported layout settings are applied. and so are layout settings are lost.

so i decided to not import them, and give a fresh new pear layout experience to user.

Reply:
i see, it is very plausible that this is a factor. 
Let's package and see the current version, I believe the removal of the workspace check fixes the issue (already merged on main) on first launch. On subsequent launch (e.g. new window), setupPearAPP will not be called, so the sidebar will not be opened, which is the desired behaviour